sci-biology/fsl/fsl-5.0.9.ebuild

```diff
@@ -0,0 +1,139 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils toolchain-funcs prefix
+
+DESCRIPTION="Analysis of functional, structural, and diffusion MRI brain imaging data"
+HOMEPAGE="http://www.fmrib.ox.ac.uk/fsl"
+SRC_URI="https://dev.gentoo.org/~jlec/distfiles/${P}-sources.tar.gz"
+
+LICENSE="FSL BSD-2 newmat"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+COMMON_DEPEND="
+	dev-libs/boost
+	media-gfx/graphviz
+	media-libs/gd
+	media-libs/glu
+	media-libs/libpng:0=
+	sys-libs/zlib
+	"
+DEPEND="${COMMON_DEPEND}"
+RDEPEND="${COMMON_DEPEND}
+	dev-lang/tcl:0=
+	dev-lang/tk:0=
+	"
+
+S=${WORKDIR}/${PN}
+
+src_prepare(){
+	epatch \
+		"${FILESDIR}/${PN}"-5.0.9-setup.patch \
+		"${FILESDIR}/${PN}"-5.0.9-headers.patch \
+		"${FILESDIR}/${PN}"-5.0.8-fsldir_redux.patch
+
+	sed -i \
+		-e "s:@@GENTOO_RANLIB@@:$(tc-getRANLIB):" \
+		-e "s:@@GENTOO_CC@@:$(tc-getCC):" \
+		-e "s:@@GENTOO_CXX@@:$(tc-getCXX):" \
+		config/generic/systemvars.mk || die
+
+	eprefixify $(grep -rl GENTOO_PORTAGE_EPREFIX src/*) \
+		etc/js/label-div.html
+
+	makefilelist=$(find src/ -name Makefile)
+
+	sed -i \
+		-e "s:-I\${INC_BOOST}::" \
+		-e "s:-I\${INC_ZLIB}::" \
+		-e "s:-I\${INC_GD}::" \
+		-e "s:-I\${INC_PNG}::" \
+		-e "s:-L\${LIB_GD}::" \
+		-e "s:-L\${LIB_PNG}::" \
+		-e "s:-L\${LIB_ZLIB}::" \
+		${makefilelist} || die
+
+	sed -e "s:\${FSLDIR}/bin/::g" \
+		-e "s:\$FSLDIR/bin/::g" \
+		-i $(grep -rl "\${FSLDIR}/bin" src/*) \
+		$(grep -rl "\${FSLDIR}/bin" etc/matlab/*)
+
+	sed -e "s:\$FSLDIR/data:${EPREFIX}/usr/share/fsl/data:g" \
+		-e "s:\${FSLDIR}/data:${EPREFIX}/usr/share/fsl/data:g" \
+		-i $(grep -rl "\$FSLDIR/data" src/*)
+
+	sed -e "s:\$FSLDIR/etc:${EPREFIX}/etc:g" \
+		-e "s:\${FSLDIR}/etc:${EPREFIX}/etc:g" \
+		-i $(grep -rl "\$FSLDIR/etc" src/*)
+
+	sed -e "s:\$FSLDIR/doc:${EPREFIX}/usr/share/fsl/doc:g" \
+		-e "s:\${FSLDIR}/doc:${EPREFIX}/usr/share/fsl/doc:g" \
+		-i $(grep -rl "\$FSLDIR/doc" src/*)
+
+	sed -i "s:\'\${FSLDIR}\'/doc:${EPREFIX}/usr/share/fsl/doc:g" \
+		$(grep -rl "\'\${FSLDIR}\'/doc" src/*)
+
+	sed -i -e "s:\$FSLDIR/etc:/etc:g" `grep -rlI \$FSLDIR/etc *`
```

Comment:
You probably want to `${EPREFIX}` in front of `/etc` in the replacement part of sed. Also keeping the `-rlI` you could probably unit this sed line with the one acting on `$(grep -rl "\$FSLDIR/etc" src/*)` surely `$(grep -rlI "\$FSLDIR/etc" *)` will cover everything you need here.